### ISSUE TITLE
Launcher icon is not fully Material.

### ISSUE THREAD

**Issue by zlsa** [anthonycr/Lightning-Browser#418]
The cloud doesn't look right with the differently sized "bulges" and the lightning looks funny IMO (I think because of the different angles). The long shadow is also too crisp and dark for Material Design.

What do you think about something like
![this](https://cloud.githubusercontent.com/assets/6366062/14939501/91f646b4-0efe-11e6-8a15-d312429463f7.png)? If you want to merge it in, I can create a PR.

**Comment by kuc** [comment]
but why gray...?

**Comment by zlsa** [comment]
Because both blue and white looked worse IMO. I can upload those versions later today.

**Comment by zlsa** [comment]
Because both blue and white looked worse IMO. I can upload those versions later today.

**Comment by zlsa** [comment]
Here are the blue and white icons:

![blue](https://cloud.githubusercontent.com/assets/6366062/14943650/35f0d470-0f93-11e6-8413-70f2bb95e1c4.png)
![white](https://cloud.githubusercontent.com/assets/6366062/14943657/4bb14ef2-0f93-11e6-8c82-25352fe169ec.png)

**Comment by dman1** [comment]
The grey has a slight winamp type look about it.  Looks good. White looks
pretty good to me also. simple and bold and light at the same time.

Less is more in my opinion on this one.

**Comment by ImAnnoying** [comment]
I love the blue one. Maybe a slightly darker shade?

**Comment by zlsa** [comment]
Here's a darker blue variant.

![blue](https://cloud.githubusercontent.com/assets/6366062/14991165/ded5b7c2-1114-11e6-8c9f-dedcfd8fb012.png)

Also, if the maintainers or @anthonycr want something else, I don't mind at all. I was just showing an example of what I believe a better icon would be.

**Comment by betsythefc** [comment]
I really like this icon redesign. The Darker blue is mu favorite.

**Comment by connyduck** [comment]
The icon really needs a redesign. The current one looks like i was from a weather app or something. The dark blue variant has my vote.

**Comment by kn00tcn** [comment]
the middle segment is quite irritating (not horizontal enough angle, possibly too wide)

diagonal shadows are such a gimmick, i dont want to keep the cycle going https://material.google.com/style/icons.html#icons-system-icons

i am wondering how it would look if the old icon simply removed the cloud & centered/enlarged the bolt

**Comment by zlsa** [comment]
Diagonal shadows are part of the spec for product icons: https://material.google.com/style/icons.html#icons-product-icons

System icons are those used on toolbars, buttons, and for "actions", not for products themselves.

I don't really mind any icon, but the current one doesn't fit in very well with most other Material apps.

**Comment by kn00tcn** [comment]
oh, i was scrolled down when i linked that

i meant to point out that not all icons have the shadow (settings, contacts rolodex, calendar, calculator, phone, etc) while a TON of third party icon sets are entirely made out of them, they ruined it

**Comment by zlsa** [comment]
Absolutely, but I think a raised lightning bolt looks better than a flat one. I can easily change it though; I don't really care as long as something eventually replaces the existing icon (with apologies to the artist who made it).
